src/shared/sleep-config.c

```diff
@@ -56,6 +58,7 @@ int parse_sleep_config(const char *verb, char ***_modes, char ***_states) {
                 { "Sleep",   "HibernateState",   config_parse_strv,  0, &hibernate_state },
                 { "Sleep",   "HybridSleepMode",  config_parse_strv,  0, &hybrid_mode  },
                 { "Sleep",   "HybridSleepState", config_parse_strv,  0, &hybrid_state },
+                { "Sleep",   "HibernateDelay",   config_parse_strv,  0, &hibernate_delay},
```

Comment:
This does not look right. It should be using a parser like `config_parse_sec`.

If you want to make minutes the default time unit, this will be a bit more work. `parse_minutes` needs to be added, similar to `parse_sec`, and then `config_parse_minutes`, similar to `config_parse_sec`. Some basic unit tests would be very much welcome.

The option should contain a suffix that indicates the default time unit: either "HibernateDelaySec=" or "HibernateDelayMinutes=".

The documentation of HibernateDelay= should then be updated, and also specify the default time unit and mention that the usual suffixes can be used (you can add a reference to systemd.time(7)).